system-tests/test/config_spec.js

```diff
@@ -236,4 +236,13 @@ describe('e2e config', () => {
       snapshot: true,
     })
   })
+
+  it('finds supportFiles in projects containing glob syntax', async function () {
```

Comment:
What do you think of pulling this test up into the launchpad? It would gain windows coverage in CI. Not sure about moving a test just due to how we run CI since this does seem like the proper place for it...

Reply:
I think an e2e test makes a lot of sense 👍 . I'll add one.